[fix] Mobile responsiveness for text

dtsivkovski opened the pull request:
Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have tested my changes locally and they work
- [x] My pull request targets the `main` branch of this repository.
- [x] I have organized any project files to the best of my ability

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

<!-- Feel free to add any additional description of changes below this line -->

![image](https://github.com/user-attachments/assets/31bdb398-6f3c-4037-8db1-ed33f28c48c0)
![image](https://github.com/user-attachments/assets/ee9a4f53-03c3-4d1a-87c7-d1b4efd117e8)

DylanDevelops reviewed (changes requested):
This seems to be a styling choice that the ShadCN accordion element inflicts on the component. Can you address this stylistic change to the actual component rather than forcing us to add this line of code for all future FAQ questions that are long?

You can find the component here: https://github.com/PantherHacks/pantherhacks.dev/blob/8cb589d10ba7ac36a59744309e3106b8fea9b93f/components/ui/accordion.tsx

Additionally, it may be worth looking at how this content is displayed for the answer as well as it might do something similarly.

Once you address this, please re-request my review and then we should be good to merge.

dtsivkovski commented:
> This seems to be a styling choice that the ShadCN accordion element inflicts on the component. Can you address this stylistic change to the actual component rather than forcing us to add this line of code for all future FAQ questions that are long?
> 
> You can find the component here: https://github.com/PantherHacks/pantherhacks.dev/blob/8cb589d10ba7ac36a59744309e3106b8fea9b93f/components/ui/accordion.tsx
> 
> Additionally, it may be worth looking at how this content is displayed for the answer as well as it might do something similarly.
> 
> Once you address this, please re-request my review and then we should be good to merge.

Just addressed this! I implemented it for the question section, and I also added a small gap between the question and the dropdown icon so there's some padding in between them. 

I looked into how the appearance looks for the answer, and it looks like the answer part of it is already aligned to the left, and looks good to me as-is.

DylanDevelops reviewed (approved):
Thank you for implementing this feedback! LGTM! 👍🏼🎉
